pkg/ext/stores/useractivity/store.go

```diff
@@ -0,0 +1,312 @@
+package useractivity
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"slices"
+	"strings"
+	"time"
+
+	ext "github.com/rancher/rancher/pkg/apis/ext.cattle.io/v1"
+	v3Legacy "github.com/rancher/rancher/pkg/apis/management.cattle.io/v3"
```

Comment:
TBH I wouldn't call it legacy. This can be misleading. `v3.Token`s are not going away for a while.

Reply:
got it, I renamed it to `v3`, and moved the old `v3` to be `v3Norman`

Reply:
@andreas-kupries fixed, thanks!